tensorboard/tools/migration/src/polymerelementify.ts

```diff
@@ -322,16 +322,20 @@ function getPropsAstNodes(props: ts.PropertyAssignment) {
           ? valueInitializer.initializer
           : undefined;
       }
+      const allowedPropertiesForComputedDecorator = ['type', 'computed'];
       const canExtractComputed =
         propProperties &&
+        propProperties.every((prop: ts.PropertyAssignment) => {
+          return allowedPropertiesForComputedDecorator.includes(propName(prop));
+        }) &&
         (propProperties.find((prop: ts.PropertyAssignment) => {
           return (
             propName(prop) === 'computed' &&
             ts.isStringLiteral(prop.initializer) &&
             // "computed" decorator is incompatible with the preperty accessor
             // or "*" for fancy diff-binding.
             parsePolymerStringDeclaration(prop.initializer.text).args.every(
-              (arg) => !arg.includes('.')
+              (arg) => !arg.includes('.') && !arg.includes('*')
```

Comment:
Presence of `.` should be enough check. `*` often supersedes a `.`. Which case is this addressing?

Reply:
That's true!  Done